bot/constants.py

```diff
@@ -661,8 +661,9 @@ class _Keys(EnvConfig, env_prefix="api_keys_"):
     "In the future, don't do that.",
     "That was a mistake.",
     "You blew it.",
-    "You're bad at computers.",
-    "Are you trying to kill me?",
+    "Application bot.exe will be closed.",
+    "Kernel Panic! *Kernel runs around in panic*",
+    "error 418. I am a teapot.",
```

Comment:
nit: only one that doesn't start with a capital letter
```suggestion
    "Error 418. I am a teapot.",
```